packages/core/src/output-formats.ts

```diff
@@ -31,6 +31,38 @@ export const sourceFormat: OutputFormat = () => (metadatas) => {
   return metadatas.map((m) => metadataToSource(m))
 }
 
+export const imgFormat: OutputFormat = () => (metadatas) => {
+  let largestImage
+  let largestImageSize = 0
+  for (let i = 0; i < metadatas.length; i++) {
+    const m = metadatas[i]
+    if ((m.width as number) > largestImageSize) {
+      largestImage = m
+      largestImageSize = m.width as number
+    }
+  }
+
+  const result: Img = {
+    src: largestImage?.src as string,
+    w: largestImage?.width as number,
+    h: largestImage?.height as number
+  }
+
+  if (metadatas.length >= 2) {
+    result.srcset = []
+    for (let i = 0; i < metadatas.length; i++) {
+      const m = metadatas[i]
+      // we don't need to create a srcset if there is only a single image
+      if (metadatas.length < 2) {
```

Comment:
`if (metadatas.length >= 2)` above makes this check unnecessary

Reply:
oops. yeah, it started there and I meant to move it out, but guess I only got the job half done :smile: thanks for catching!